ArcGISRuntimeSDKQt_CppSamples/Scenes/TerrainExaggeration/TerrainExaggeration.cpp

```diff
@@ -0,0 +1,68 @@
+// [WriteFile Name=TerrainExaggeration, Category=Scenes]
+// [Legal]
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#include "TerrainExaggeration.h"
+
+#include "ArcGISTiledElevationSource.h"
+#include "Scene.h"
+#include "SceneQuickView.h"
+
+using namespace Esri::ArcGISRuntime;
+
+TerrainExaggeration::TerrainExaggeration(QQuickItem* parent /* = nullptr */):
+  QQuickItem(parent)
+{
+}
+
+void TerrainExaggeration::init()
+{
+  // Register classes for QML
+  qmlRegisterType<SceneQuickView>("Esri.Samples", 1, 0, "SceneView");
+  qmlRegisterType<TerrainExaggeration>("Esri.Samples", 1, 0, "TerrainExaggerationSample");
+}
+
+void TerrainExaggeration::componentComplete()
+{
+  QQuickItem::componentComplete();
+
+  // Create a scene and give it to the SceneView
+  m_sceneView = findChild<SceneQuickView*>("sceneView");
+  Scene* scene = new Scene(Basemap::nationalGeographic(this), this);
```

Comment:
is this the same basemap used in the design? I believe the other samples may use a different basemap. If so, please update and update the screenshots

Reply:
Java (which is what I was using as a reference) uses natGeo basemap, iOS/macOS use streets. Which one do you prefer?